src/components/Header/HeaderContact.js

```diff
@@ -55,7 +56,7 @@ const HeaderContact = ({
           style={[
             styles.text,
             subtitleStyle,
-            {color: disabledNextBtn ? COLORS.gray6 : COLORS.blue}
+            {color: disabledNextBtn ? COLORS.gray6 : colors.darkBlue}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated correctly from `COLORS.blue` to `colors.darkBlue`. However, `COLORS.gray6` is still being used. To maintain consistency, it should also be updated to use the new `colors` import.
```diff
- {color: disabledNextBtn ? COLORS.gray6 : colors.darkBlue}
+ {color: disabledNextBtn ? colors.gray6 : colors.darkBlue}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->